explore/src/main/scala/explore/constraints/TimingWindowsPanel.scala

```diff
@@ -0,0 +1,398 @@
+            )
+          ).withRef(resize.ref),
+          current
+            .map { e =>
+              <.div(
+                ExploreStyles.TimingWindowEditor,
+                <.span(
+                  "Open on ",
+                  Datepicker(onChange =
+                    (newValue, _) =>
+                      newValue.fromDatePickerToZDTOpt.foldMap { i =>
+                        selectedStartsOn.set(i.withSecond(0).withNano(0))
+                      }
+                  )
+                    .showTimeInput(true)
+                    .selected(e.startsOn.toDatePickerJsDate)
+                    .dateFormat("yyyy-MM-dd HH:mm"),
+                  <.span(" UTC and:")
+                ),
+                <.div(
+                  RadioButton("forever",
+                              id = "forever-option",
+                              checked = e.openForever,
+                              onChange = (_, checked) => selectedTW.mod(_.toForever).when_(checked)
+                  ),
+                  <.label("Remain open forever", ^.htmlFor := "forever-option")
+                ),
+                <.div(
+                  RadioButton(
+                    "close-on",
+                    id = "close-on-option",
+                    checked = e.closeOn,
+                    onChange = (_, checked) =>
+                      CallbackTo.now
+                        .flatMap(i =>
+                          selectedTW.mod(_.toCloseOn(ZonedDateTime.ofInstant(i, Constants.UTC)))
+                        )
+                        .when_(checked)
+                  ),
+                  <.label("Close on ", ^.htmlFor := "close-on-option"),
+                  selectedCloseOn.asView.map { closeOn =>
+                    React.Fragment(
+                      Datepicker(onChange =
+                        (newValue, _) =>
+                          newValue.fromDatePickerToZDTOpt.foldMap { i =>
+                            selectedCloseOn.set(i.withSecond(0).withNano(0))
+                          }
+                      )
+                        .showTimeInput(true)
+                        .selected(closeOn.get.toDatePickerJsDate)
+                        .dateFormat("yyyy-MM-dd HH:mm"),
+                      <.span(" UTC")
+                    )
+                  }
+                ),
+                <.div(
+                  ExploreStyles.TimingWindowRemainOpen,
+                  RadioButton(
+                    "remain-open",
+                    id = "remain-open",
+                    checked = e.remainOpenFor,
+                    onChange = (_, checked) =>
+                      selectedTW
+                        .mod(_.toRemainOpen(NonNegDuration.unsafeFrom(Duration.ofDays(2))))
+                        .when_(checked)
+                  ),
+                  <.label("Remain open for", ^.htmlFor := "remain-open"),
+                  selectedRemainOpenFor.asView.map { remainOpen =>
+                    FormInputEV(
```

Comment:
It seems like you have a mix of SUI and primereact. Could you use FormInputTextView here?

Reply:
I noticed, I wasn't sure if we had a prime react version